pyaedt/hfss.py

```diff
@@ -6292,34 +5845,32 @@ def lumped_port(
         ...                                "BoxLumped1","copper")
         >>> box2 = hfss.modeler.create_box([0, 0, 60], [10, 10, 5],
         ...                                "BoxLumped2", "copper")
-        >>> hfss.lumped_port("BoxLumped1", "BoxLumped2",
-        ...                  hfss.AxisDir.XNeg, 50,
-        ...                  "LumpedPort", True, False)
+        >>> hfss.lumped_port("BoxLumped1","BoxLumped2",hfss.AxisDir.XNeg,50,"LumpedPort",True,False)
         PyAEDT INFO: Connection Correctly created
         'LumpedPort'
 
         """
         if create_port_sheet:
-            signal = self.modeler.convert_to_selections(signal)
+            assignment = self.modeler.convert_to_selections(assignment)
             reference = self.modeler.convert_to_selections(reference)
-            if not self.modeler.does_object_exists(signal) or not self.modeler.does_object_exists(reference):
+            if not self.modeler.does_object_exists(assignment) or not self.modeler.does_object_exists(reference):
                 self.logger.error("One or both objects do not exist. Check and retry.")
                 return False
             sheet_name, point0, point1 = self.modeler._create_sheet_from_object_closest_edge(
-                signal, reference, integration_line, port_on_plane
+                assignment, reference, integration_line, port_on_plane
             )
         else:
-            if isinstance(signal, list):
-                objs = self.modeler.get_faceid_from_position(signal)
+            if isinstance(assignment, list):
+                objs = self.modeler.get_faceid_from_position(assignment)
                 if len(objs) == 1:
-                    signal = objs[0]
+                    assignment = objs[0]
                 elif len(objs) > 1:
-                    self.logger.warning("More than 1 face found. Getting first.")
-                    signal = objs[0]
+                    self.logger.warning("More than one face was found. Getting the first one.")
+                    assignment = objs[0]
                 else:
                     self.logger.error("No Faces found on given location.")
                     return False
-            sheet_name = self.modeler.convert_to_selections(signal, False)
+            sheet_name = self.modeler.convert_to_selections(assignment, False)
             if isinstance(integration_line, list):
                 if len(integration_line) != 2 or len(integration_line[0]) != len(integration_line[1]):
                     self.logger.error("List of coordinates is not set correctly")
```

Comment:
```suggestion
                    self.logger.error("List of coordinates is not set correctly.")
```
Strings should have concluding punctuation.